addOns/ascanrulesAlpha/src/main/resources/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties

```diff
@@ -33,4 +33,9 @@ ascanalpha.mongodb.extrainfo.authbypass=More probably a MongoDB attack has bypas
 
 ascanalpha.forbiddenBypass.name=Bypassing 403
 ascanalpha.forbiddenBypass.desc=Bypassing 403 endpoints may be possible, the scan rule sent a payload that caused the response to be accessible (status code 200).
-ascanalpha.forbiddenBypass.refs=https://www.acunetix.com/blog/articles/a-fresh-look-on-reverse-proxy-related-attacks/\nhttps://i.blackhat.com/us-18/Wed-August-8/us-18-Orange-Tsai-Breaking-Parser-Logic-Take-Your-Path-Normalization-Off-And-Pop-0days-Out-2.pdf\nhttps://www.contextis.com/en/blog/server-technologies-reverse-proxy-bypass
\ No newline at end of file
+ascanalpha.forbiddenBypass.refs=https://www.acunetix.com/blog/articles/a-fresh-look-on-reverse-proxy-related-attacks/\nhttps://i.blackhat.com/us-18/Wed-August-8/us-18-Orange-Tsai-Breaking-Parser-Logic-Take-Your-Path-Normalization-Off-And-Pop-0days-Out-2.pdf\nhttps://www.contextis.com/en/blog/server-technologies-reverse-proxy-bypass
+
+ascanalpha.webCacheDeception.name=WebCacheDeception.
+ascanalpha.webCacheDeception.desc=WebCacheDeception may be possible.It may be possible for unauthorised user to view sensitive data on this page.
```

Comment:
```suggestion
ascanalpha.webCacheDeception.desc=Web cache deception may be possible.It may be possible for unauthorised user to view sensitive data on this page.
```